### DISCUSSION TITLE
The Quest for the Golden Soroban Metaverse

### DISCUSSION THREAD

**Discussion by stanpilev** [stellar/sorobanathon#146] [General]
Hi! We want to thank you for stopping by! 

You can now enter the Stellar Soroban Metaverse experience. We create everything from scratch and build the experience on Unreal Engine. This land will be used in our metaverse when it releases next year and we will continue to work on it so that people can actually go through all the chapters to learn, interact and play. 
[The Golden Soroban Metaverse](https://www.besmetaverse.com/stellar-soroban)

We minted 7 NFTs by the chapters of the journey on our official Litemint account. 

Chapter 0 
https://litemint.com/items/GCKHHE33Q64N7NVM2AEWLWK4T55VPZBFQ5WNU7E3TNO6LTGM6JR3GJGG/0
Chapter 1
https://litemint.com/items/GCUEVXQ4QUXZY5OOHWLFFASMV7WRTKT2QMOK5QHFRWN7XZTCVCFYF6XK/1
Chapter 2
https://litemint.com/items/GAP6QH4R4H3JSH4KKL7N6GBOXCLJKH766X4HNSBORIYBYMGHUPZ55IWH/2
Chapter 3
https://litemint.com/items/GDLFQTXU2KN4Q6XPX73TVAG5T27QG67YSA34GOL7FHIIBPV3WTUGDAON/3
Chapter 4
https://litemint.com/items/GAMCKMPEKBAZ47WYHENZB76WQ6VTFZFU4ZN3EN2J5KVBONBSBFLFIDGK/4
Chapter 5
https://litemint.com/items/GDZSXAIB6ZJOQACHKNHEI3UXCLEZ5USH7DE4THKESJBORHL3WEHIG3HM/5
Chapter 6
https://litemint.com/items/GBRVVGU5XH3JM4ULQISVFM4CU5GF24VVI6GBWRCRMQZVUJM4KQ7J274J/6

**Comment by stanpilev** [comment]
Our Twitter account is https://twitter.com/BESmetaverse

**Reply by stanpilev** [reply]
![Screenshot (391)](https://user-images.githubusercontent.com/59807372/207983608-7a847d30-3ed9-4222-897a-f01d88e5a971.png)
![Screenshot (389)](https://user-images.githubusercontent.com/59807372/207983629-f88f0827-7522-435e-83df-386ff50d3bc8.png)
![Screenshot (388)](https://user-images.githubusercontent.c
![Screenshot (386)](https://user-images.githubusercontent.com/59807372/207983701-fb367113-75e0-4ec0-ba59-8e107b1ba8f7.png)
om/59807372/207983652-dd254af6-3c3a-401c-a85a-049c8d9a6eb2.png)
![Screenshot (387)](https://user-images.githubusercontent.com/59807372/207983675-06738ad2-acc6-4292-a1db-7091e0c55c77.png)

**Comment by Xicaaa** [comment]
Let's goo!

**Comment by Stellarhamza** [comment]
Really liked the locations, amazing stuff the user experince in game is amazing as well.

**Comment by stanpilev** [comment]
https://user-images.githubusercontent.com/59807372/207726139-fc75db9d-f172-4a0d-9b81-ced5a7c64416.mov

A bit low quality as we can add only a 10Mb file here. The best experience is to go directly to the metaverse link https://www.besmetaverse.com/stellar-soroban

**Reply by OliaXLM** [reply]
Mind blowing! 😯

**Reply by stanpilev** [reply]
thank you!

**Comment by mgaylor** [comment]
Selfie!
![bee_screenshot](https://user-images.githubusercontent.com/115509084/208034512-07fcdc51-18dc-4169-a12b-20d90ee75a4b.png)

**Reply by stanpilev** [reply]
Looking good! Thank you for testing out 🔥

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
